docs/docs/general/07-Troubleshooting/deprecated-endpoints.md

```diff
@@ -0,0 +1,15 @@
+# Deprecated Endpoints
+
+The askui library is sending requests with screenshots and/or instructions
+to endpoints of our inference API to get annotations and commands.
+We are starting to use our new `inference` endpoint for this kind of requests.
+Earlier versions (`v0.3.2` and prior) are using the `predict-comand` and `annotate` endpoint
+of the API.
+The old endpoints are still working but we are not planning to maintain them.
+Both endpoints will be deprecated in the future.
+
+Please upgrade your askui version to use our new endpoint if you are using askui `v0.3.2` and earlier:
+
+```shell
+npm update askui
```

Comment:
Was the problem solved that caused trouble sometimes when updating like this?

Reply:
good catch, 
yes this might not work. 
you can change it to  `npm install askui@latest`

Reply:
had also probles with update but Samirs suggestion works fine :+1: